pyasice/tests/test_container.py

```diff
@@ -110,8 +144,21 @@ def test_bdoc_finalize(temporary_file, xml_signature_rsa_signed):
 
     bdoc_file2 = Container(buffer)
 
-    assert bdoc_file2._enumerate_signatures() == ["META-INF/signatures1.xml"]
+    assert bdoc_file2.signature_file_names == ["META-INF/signatures1.xml"]
     assert bdoc_file2.data_file_names == ["test.txt"]
 
     bdoc_file2.verify_signatures()
     assert "No exception was raised"
+
+
+def test_bdoc_prepare_signature(certificate_rsa_bytes):
```

Comment:
Seems to be a low hanging fruit to add a test here that does the same but passes certificate instance instead of certificate bytes, similarly how we test above different ways to initialize the container.

Not sure how useful it would be :thinking: